src/transform/lib.rs

```diff
@@ -234,7 +264,7 @@ impl Optimizer {
         mut relation: RelationExpr,
         indexes: &HashMap<GlobalId, Vec<Vec<ScalarExpr>>>,
     ) -> Result<OptimizedRelationExpr, TransformError> {
-        self.transform(&mut relation, indexes)?;
+        self.transform(&mut relation, &indexes)?;
```

Comment:
What's this about? Looks like a double `&` but maybe it makes sense (not a huge deal, but it stuck out in the diff).

Reply:
Hm, not sure what that was about!